internal/admission/defaulting_test.go

```diff
@@ -57,7 +57,7 @@ func TestTimeout(t *testing.T) {
 			After(timeout/2).
 			Return(validate.Valid, nil).Once()
 		defer validationSvc.AssertExpectations(t)
-		webhook := NewDefaultingWebhook(client, validationSvc, timeout, logger.Sugar())
+		webhook := NewDefaultingWebhook(client, validationSvc, timeout, false, logger.Sugar())
```

Comment:
We can move it to const: `TurnOffStrictMode`

Reply:
It is moved to const: `StrictModeOff` and also added `StrictModeOn` for the next test.